test/blueprints/Do.Test.Blueprints.Service.Test/IntegrationSpec.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Do.Architecture;
+using Microsoft.AspNetCore.Mvc.Testing;
+
+namespace Do.Test;
+
+public abstract class IntegrationSpec
+{
+    static WebApplicationFactory<TestProgram> _factory = default!;
+
+    static IntegrationSpec()
+    {
+        Environment.SetEnvironmentVariable("ASPNETCORE_ENVIRONMENT", "Development");
+
+        _factory = new WebApplicationFactory<TestProgram>();
+    }
+
+    public static void Run() =>
+        Forge.New
+            .Service(
+               business: c => c.Default(),
+               database: c => c.Sqlite(),
+               exceptionHandling: ex => ex.Default(typeUrlFormat: "https://do.mouseless.codes/errors/{0}"),
+               configure: app => app.Features.AddConfigurationOverrider()
+           )
+        .Run();
```

Comment:
inline this directly to `Main`

Reply:
After refactoring every `IntegrationSpec` implements its own `Run()` thus I wasn't able to do this change either